node-graph/gcore/src/graphic_element/renderer.rs

```diff
@@ -25,9 +25,15 @@ use vello::*;
 pub struct ClickTarget {
 	pub subpath: bezier_rs::Subpath<PointId>,
 	pub stroke_width: f64,
```

Comment:
I think it would be wise to make these private and force their manipulation to go through the API in order to keep the bounding box from getting outdated.